tests/functional/modules/test_zos_job_query_func.py

```diff
@@ -30,7 +30,7 @@ def test_zos_job_query_func(ansible_zos_module):
     results = hosts.all.zos_job_query(job_name="*", owner="*")
     pprint(vars(results))
     for result in results.contacted.values():
-        assert result.get("changed") is False
+        # assert result.get("changed") is False
```

Comment:
Why was this assertion commented? If we are just querying the result of a just changed should be false right?

Reply:
I'm going to re-run without the comment (so the assertion is tested).  I checked the code, and it should always return false, so this should be a trivial check.

Reply:
There was a problem with the 'skip tests' feature, so I've fixed that and run just the relevant test functions:

![image](https://github.com/ansible-collections/ibm_zos_core/assets/39140439/cd8f98f7-db8a-4593-bcf4-6a9449ae7582)

It passed with the assertion now un-commented.  Thanks for catching that!

Reply:
Note, this passed with the following tests in skip_tests field

functional/modules/test_module_security.py
functional/modules/test_zos_copy_func.py
functional/modules/test_zos_encode_func.py
functional/modules/test_zos_fetch_func.py
functional/modules/test_zos_data_set_func.py

There appears to be something out-of-whack with read/write file systems.